### DISCUSSION TITLE
running ileapp GUI

### DISCUSSION THREAD

**Discussion by huddledfiber** [abrignoni/iLEAPP#405] [Q&A]
It's been a while, but I have successfully used the ileappGUI.py on my machine.  When I tried today I'm getting these errors.

Traceback (most recent call last):
  File "C:\iLEAPP-master\ileappGUI.py", line 4, in <module>
    import ileapp
  File "C:\iLEAPP-master\ileapp.py", line 6, in <module>
    import scripts.report as report
  File "C:\iLEAPP-master\scripts\report.py", line 10, in <module>
    from scripts.ilapfuncs import logfunc
  File "C:\iLEAPP-master\scripts\ilapfuncs.py", line 12, in <module>
    import simplekml
ModuleNotFoundError: No module named 'simplekml'

I updated python to newest version (3.11) and downloaded and extracted the newest version of ileapp from Github.  Still getting errors.

**Comment by abrignoni** [comment]
3.11 is not supported currently.

ALEAPP and RLEAPP do support 3.10 & 3.11. Don't forget to pip install -r
requirements for each Python version you are running in your system.

**Reply by huddledfiber** [reply]
I have 3.8 as well.  That should work then?

**Comment by abrignoni** [comment]
Yes. Make sure you do pip3.8 install -r requirements.txt

**Reply by huddledfiber** [reply]
Awesome, thank you!
